src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -10,13 +10,39 @@
     [quo.foundations.gradients :as gradients]
     [quo.theme]
     [react-native.core :as rn]
+    [react-native.reanimated :as reanimated]
     [schema.core :as schema]
+    [utils.datetime :as datetime]
     [utils.i18n :as i18n]))
 
+(def timing-options-out 650)
+(def timing-options-in 1000)
+
+(defn on-load-end
+  [{:keys [load-time set-state loader-opacity image-opacity]}]
+  (reanimated/animate-shared-value-with-timing loader-opacity 0 timing-options-out :easing1)
+  (reanimated/animate-shared-value-with-timing image-opacity 1 timing-options-in :easing1)
+  (if (> load-time 200)
+    (js/setTimeout
+     (fn []
+       (set-state (fn [prev-state]
+                    (assoc prev-state :image-loaded? true))))
+     500)
+    (set-state (fn [prev-state]
+                 (assoc prev-state :image-loaded? true)))))
+
+(defn on-load-error
+  [set-state]
+  (js/setTimeout (fn []
+                   (set-state (fn [prev-state] (assoc prev-state :image-error? true))))
+                 800))
+
 (defn- fallback-view
-  [{:keys [label theme]}]
-  [rn/view
-   {:style (style/fallback {:theme theme})}
+  [{:keys [label theme image-opacity]}]
+  [reanimated/view
+   {:style (reanimated/apply-animations-to-style
+            {:opacity image-opacity}
+            (style/fallback {:theme theme}))}
```

Comment:
Our guideline for `apply-animations-to-style` is to use it in the `style` file

Also we made a recent update to support passing an array of styles instead of using `apply-animations-to-style` https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#pass-a-vector-of-styles-to-reanimated-view

Reply:
@J-Son89 I see, but are we still using the function `apply-animations-to-style` tho? What I currently see in the guidelines is we pass an array of styles: 
```
     {:style [{:opacity   opacity
               :transform [{:translate-x scroll-x}]}
              {:flex-direction :row}]}
```

Reply:
aha, so you mean just remove the use of `apply-animations-to-style ` completely? 
That sounds better if so 👍

Reply:
@J-Son89 @OmarBasem @Parveshdhull Yeah, we need to update the guidelines and rename the animated functions to add the prefix `use-`


> @OmarBasem - imo this guideline is most likely wrong and a bad idea. As @ulisesmac pointed out recently, this function `apply-animations-to-style` is actually using a hook and should really be called `use-animated-styles` or something of this sort. Hiding this in the style file is likely to lead to bugs (see @ajayesivan's comment below) and we should keep it in the view/ perhaps reconsider the format in how we pass these styled values.

Reply:
> aha, so you mean just remove the use of `apply-animations-to-style ` completely? That sounds better if so 👍

@J-Son89 Be careful, actually the vector syntax doesn't behave exaclty the same in all cases. I've learnt by experience that it requires a default value in the regular styles section, otherwise sometimes the animation looks inconsistent at the beginning, this is only needed if your animation doesn't behave as expected:

Example, if we want to animate the opacity, you should provide the "initial" opacity in the regular styles:
```clojure
[reanimated/view
 {:style [;; Regular styles:
          {,,,
           :opacity 0 ;; This is the opacity at the beginning of the animation
           ,,,}
          ;; Animated styles:
          {,,,
           :opacity my-shared-value
           ,,,}
          ]}]
```

Reply:
added a default value to each style using opacity. all looks good now 👍